repos/ports/run/vbox_win.inc

```diff
@@ -74,7 +74,7 @@ append config_of_app {
 		needs to be enabled in the .vbox file (disabled by default).
 		(Recording is configured to use the external mic.)
 	-->
-	<start name="audio_drv">
+	<start name="audio_drv" priority="-1">
```

Comment:
The choppyness is not eliminated completely but the quality of the output is improved. The issue was discovered when testing audio on hw_x86_64_muen/Virtualbox and @cnuke suggested the change until he has time to investigate the underlying cause. Maybe @cnuke can comment if it still should be applied.